### ISSUE TITLE
UE 4.27 / 5 support?

### ISSUE THREAD

**Issue by jfaz13** [proletariatgames/unreal.hx#143]
Hello,

I'd like to use this in my project but it seems the API has changed a decent bit since latest supported version. I was wondering if there are any updates on the horizon to add UE 4.27 or UE 5 support as this is a really nice project.

Thank you

**Comment by lublak** [comment]
Im also interested but i don't think this project is supported anymore.

**Comment by tgienger** [comment]
I was getting excited about this until I saw the dates on the latest changes :(

Hopefully someone will pick this up and continue to work on it. I'd love to use haxe in unreal.

**Comment by lublak** [comment]
@tgienger I hope it to! I like haxe and I like unreal. So unrealhx ... Jear I like it too :P
Also is haxe easier to use. I try to figure out how they created unrealhx but it's a big chunk of code.

**Comment by TheWitheredStriker** [comment]
Yeah, I agree with all of you. If I ever get good enough at coding to do so, I'll fork the project myself. This is too good to die.

**Comment by geeede** [comment]
It would be cool if they will continue to support new Unreal releases like 4.27.
Too bad, the repo seams dead.

**Comment by jfaz13** [comment]
Yeah it seems someone needs to fork this project and take it over. They have an extern generator:
https://github.com/proletariatgames/UnrealHxGenerator
might be worth looking into if anyone decides to take it into their own hands.

**Comment by TheWitheredStriker** [comment]
I would love to be that person, but I don't currently have the time or skill to lead such an endeavour right now. That said, if no one else has launched a fork when I've familiarized myself with C++, Haxe and UE, I'd be down to do it.

**Comment by ashifolfi** [comment]
I'm currently working on trying to get this plugin working in 4.27 for a project I've been working on. I've made good progress and am basically just stuck at building right now. I've forked the project and will publish my changes if I manage to get it working.

**Comment by TheWitheredStriker** [comment]
Awesome! Keep us posted!

**Comment by ashifolfi** [comment]
> I'm currently working on trying to get this plugin working in 4.27 for a project I've been working on. I've made good progress and am basically just stuck at building right now. I've forked the project and will publish my changes if I manage to get it working.

Unfortunately I have hit a wall I am unable to break past in this journey. The entire build process is littered with outdated, deprecated, incorrect syntax that I don't have the knowledge to fix. I wish I had something worthwhile to share but unfortunately that is not the case.
